packages/elements/src/slider/__test__/slider.test.js

```diff
@@ -464,4 +474,15 @@ describe('slider/Slider', function () {
       expect(el.stepRange).to.equal(5.5);
     });
   });
+
+  describe('Marker', function () {
+    it('Set position of marker correctly when new marker added', async function () {
+      const el = await fixture('<ef-slider></ef-slider>');
```

Comment:
Mocha creates a new Slider for each test case with `beforeEach()` already. No need to create another one here.